### PR TITLE
Invite rewards tweaks for v2

### LINKED ISSUE #2203: [Invites v2] Add entry point for sending invites
### Description

[Designs](https://www.figma.com/file/DL6Huw867n1EZOVPJ2kNmp/Invite-Rewards?node-id=1195%3A24849)

In the invites v1 program, invites and the send flow were tied together. It would be nice to split them out if the engineering effort is not too high. One advantage is you no longer have to send funds to invite someone.

### Acceptance Criteria
-[] Be able to select participants based on region

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

Tweak the invite rewards v1 program to work for v2:
- reuse everything from v1 including the remote config variables (no reason to add new ones since the programs will not co-exist)
- add home card for invites. the priority is unchanged from v1, and should come after backup/supercharge/payment request
- update translations

Screenshot 2022-03-25 at 10 18 33

To come:
- maybe update website link
- maybe update home card image

### Other changes

N/A

### Tested

Manually, existing unit tests

### How others should test

When the remote config variable "inviteRewardsEnabled" is true, you should see a home card for invite rewards. On the send screen, you should see the invite rewards banner.

### Related issues

- Fixes #2203 

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/home/NotificationBox.tsx [hunk 1]
```diff
@@ -145,6 +149,29 @@ function useSimpleActions() {
     })
   }
 
+  if (inviteRewardsEnabled) {
```
Comment (gnardini) [inline packages/mobile/src/home/NotificationBox.tsx:152]: Just to confirm, this card is not dismissable?

**Review by gnardini — APPROVED** [review]
🙌

File: packages/mobile/src/home/NotificationBox.tsx (hunk 1 above)
Comment (kathaypacific) [reply packages/mobile/src/home/NotificationBox.tsx:152]: lol yeah 😅 i think this experiment is going to be super short (a few days) so hopefully not a big issue. i'll double check though!

**Comment by kathaypacific** [comment]
Closing this as we will create the home cards from admin dash https://valora-app.slack.com/archives/C038A310CEL/p1648223013032869?thread_ts=1648198994.077819&cid=C038A310CEL

**Pull request closed without merging**